docs/GettingStartedDocs/Contributors/ExperimentalSupportRHEL8.md

```diff
@@ -0,0 +1,122 @@
+# Experimental - Getting Started with Open Enclave on RHEL8
+
+## Platform requirements
+
+- RHEL8
+
+Please refer to [SGXSupportLevel.md](/docs/GettingStartedDocs/SGXSupportLevel.md) to determine the SGX support level for your target system. If your system does not have SGX1 or SGX1+FLC support, you can use the simulation mode.
+
+Note: Attestation is not supported on RHEL8.
+
+## Clone Open Enclave SDK repo from GitHub
+
+Install git.
+
+```bash
+sudo yum install git
+```
+
+Use the following command to download the source code (make sure `git` is installed before doing this):
+
+```bash
+git clone https://github.com/openenclave/openenclave.git
+```
+
+This creates a source tree under the directory called openenclave.
+
+## Install project requirements
+
+First, change directory into the openenclave repository:
+
+```bash
+cd openenclave
+```
+
+Ansible is required to install the project requirements. If not already installed, you can install it by running:
+
+```bash
+sudo scripts/ansible/install-ansible.sh
+```
+
+On a system which supports SGX1 or SGX+FLC, use the command below to set up prerequisites. Please note that the Open Enclave SDK does not support attestation on RHEL as yet.
+
+```bash
+ansible-playbook scripts/ansible/oe-contributors-setup-sgx1.yml
+```
+
+On a system which does not support SGX1 or SGX1+FLC, use the command below to set up prerequisites. You would need to run tests in simulation mode.
+
+```bash
+ansible-playbook scripts/ansible/oe-contributors-setup.yml
```

Comment:
On RHEL oe-contributors-setup.yml will fail:
1. Without sgx hardware: the script will fail because of hardware check.
2. Witth sgx hardware: there is no dirver rpm available(open source dirver is avaible)
So for non-sgx system user can still install through oe-contributors-setup-sgx1.yml, then rum in simulation mode.

Reply:
Thanks @nonpolarity